libs/designer-ui/src/lib/connectors/predicates.ts

```diff
@@ -22,3 +29,14 @@ export const isCustomConnector = (connector: Connector | OperationApi | string):
   const connectorId = isString(connector) ? connector : connector.id;
   return isCustomConnectorId(connectorId);
 };
+
+export const isPremiumConnector = (connector: Connector | OperationApi | string): boolean => {
+  const hostIsCustomConnectorFn = HostService()?.isPremiumConnector;
+  if (hostIsCustomConnectorFn) {
```

Comment:
if This isn't defined, please return false immediately. This will prevent possible behavior changes to Logic Apps or in ICM Automation where premium connectors don't exist.